.eslintrc.json

```diff
@@ -36,6 +36,7 @@
 	"rules": {
 		"array-bracket-spacing": [ "error", "always" ],
 		"brace-style": [ "error", "1tbs" ],
+		"camelcase": ["error", { "properties": "never" }],
```

Comment:
Missing spaces here :) `[ "error", { "properties": "never" } ]`.

We should run eslint against .eslintrc.json :)